src/index.d.ts

```diff
@@ -0,0 +1,18 @@
+
+interface Props<T> extends DOMAttributes<T> {
+    children: JSX.Element;
+    component?: string;
+    duration?: number;
+    disableInitialAnimation?: boolean;
+    className: string;
```

Comment:
Thanks for the PR! `className` should be optional I think

Reply:
I think so too. Will update that.